examples/gno.land/p/demo/audio/biquad/biquad.gno

```diff
@@ -0,0 +1,57 @@
+package biquad
+
+import "math"
+
+type Filter struct {
+	a1, a2, b0, b1, b2     float64
+	x1_f, x2_f, y1_f, y2_f float64
```

Comment:
why the _f?

Reply:
I initially had code for forward "f" and backward propagation, its unnecessary here so I removed it in 54c8827d